M2/Macaulay2/packages/gfanInterface.m2

```diff
@@ -206,6 +206,9 @@ MarkedPolynomialList.synonym = "marked polynomial list";
   -- the monomial inL#i is the marked monomial, which
   -- should occur with the same coefficient in L#i.
 
+MarkedPolynomialList == MarkedPolynomialList := (L1, L2) ->
+	set transpose L1 === set transpose L2
+
```

Comment:
The word "List" in "MarkedPolynomialList" implies that for objects of this type, the order matters, so introducing an equality test like this into the package will result in cognitive dissonance.  Could you instead just fix the test, so it does the appropriate thing?

Reply:
Makes sense -- will do!